src/test/scala/com/github/pjfanning/pekkobuild/PekkoHttpDependency.scala

```diff
@@ -0,0 +1,20 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, which was derived from Akka.
+ */
+
+/*
+ * Copyright (C) 2017-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package com.github.pjfanning.pekkobuild
+
+object PekkoHttpDependency extends PekkoDependency {
```

Comment:
I'm still a bit skeptical about whether we keep this code out of the `main` source. I think this is still a build tool and for our convenience. I don't see the reason to be so protective. I would like this to the changes that you added to `sbt` that added Apache specific resolvers - that only a small number of active Apache projects would care about.

Reply:
The thing is that these changes are not specific per project so it doesn't make sense to force every project to contain them, the changes added in sbt however are (and they are also generic resolvers which are useful to projects even outside of Pekko).

Also as I pointed out earlier, its actually **_wrong_** to place an actual instance of `PekkoHttpDependency` in the `main` source because its going to differ from project to project i.e. `minVersion` (if set) is going to differ from one Pekko module to the next, and if we are complaining about the boilerplate values such as `currentVersion` and `checkProject`, they can be auto derived (I just didn't want to do it in this PR because it would make the PR too big).

In any case the main goal that this project is meant to solve is to not replicate the large amount of boilerplate (i.e. `trait PekkoDependency`), adding a single `.scala` file that is 3-4 lines long is not the real issue here.